int64[] start_pos
int64[] start_vel
int64[] goal
string dmp_name